example/build.rs

```diff
@@ -1,3 +1,6 @@
 fn main() {
-    tauri_build::build()
+    println!("cargo:rustc-link-arg=-Wl,-rpath,@executable_path/../Resources/pyembed/lib");
+    println!("cargo:rustc-link-arg=-Wl,-rpath,@executable_path/./pyembed/lib");
```

Comment:
Why are there two different locations for `pyembed/lib`?

Reply:
One for running with cargo run, and the other for the final executable location in the dmg.

Reply:
When using `cargo run` for development, I recommend using the system's Python. python-build-standalone has some quirks that might make developers' lives difficult.